sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileSystemClient.cs

```diff
@@ -2378,5 +2378,241 @@ private void SetNameFieldsIfNull()
             return sasUri.ToUri();
         }
         #endregion
+
+        #region Get Deleted Paths
```

Comment:
Note the limit amount of properties we are returning for the paths in `GetDeletedPaths()`.  I was told by the FE team to try to stick with delete-related properties only.

There are probably a lot of properties we could pull from the swagger here - https://github.com/Azure/azure-rest-api-specs/pull/11741

Reply:
- Note that I copied BlobContainerClient.GetBlobsHierarchical(). 
- GetDeletedPaths() doesn't currently support the delimiter parameter, but it will in the future.
- I didn't want to create to APIs for GetDeletedPaths(), I thought it was unnecessary.